e2e_tests/tests/cluster/test_task.py

```diff
@@ -72,3 +73,34 @@ def test_task_get_config() -> None:
         """
         )
         assert sorted(expected.items()) == sorted(result.items())
+
+@pytest.mark.e2e_cpu
+def test_task_fork() -> None:
+    sess = api_utils.determined_test_session(admin=True)
+    with open("e2e_tests/test_config.yaml", "r") as config_file:
+        config = command.parse_config(config_file, None, [], [])
+        config_text = util.yaml_safe_dump(config)
+        context_directory = context.read_v1_context(Path("e2e_tests"), [])
+        req = bindings.v1CreateGenericTaskRequest(
+            config=config_text,
+            contextDirectory=context_directory,
+            projectId=None,
+            forkedFromId=None
+        )
+        task_resp = bindings.post_CreateGenericTask(sess, body=req)
+        assert len(task_resp.taskId) > 0
+        config_file.close()
+    
+        with open("e2e_tests/test_config_fork.yaml", "r") as config_file:
+            config = command.parse_config(config_file, None, [], [])
+            config_text = util.yaml_safe_dump(config)
+            context_directory = context.read_v1_context(Path("e2e_tests"), [])
+            req = bindings.v1CreateGenericTaskRequest(
+                config=config_text,
+                contextDirectory=context_directory,
```

Comment:
just want to make sure: we shouldn't require the user to pass the context files (or config) again when forking: the idea is that these will be inherited from the parent run.

Reply:
Updated: the api should now support forking without provided child config file. However, since the `det create` cli command requires config files as input. I created a new command `det fork` just for forking without a child config and context directory.